bmiapp/lib/pageinfo.dart

```diff
@@ -37,14 +30,10 @@ class PageinfoState extends State {
             decoration: const InputDecoration(hintText: "Enter Weight in kg"),
             onChanged: (value) {
               setState(() {
```

Comment:
For dynamic effect onchanged is used to change images on the fly, but it could lead to performance issues to changing to response only on button could help.